topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -172,7 +170,6 @@ private NavbarBrand createBadge(String headlineText) {
 	 */
 	private Image createLogoImage(String iconUrl) {
 		Image icon = new Image(ifPresent(iconUrl, "images/broken.png"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "icon". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1488)